modules/@apostrophecms/asset/index.js

```diff
@@ -37,12 +47,48 @@ module.exports = {
           await fs.mkdirp(buildDir);
           await fs.remove(modulesDir);
           await fs.mkdirp(modulesDir);
-          await fs.remove(bundleDir);
-          await fs.mkdirp(bundleDir);
+
+          let rebuildAposUi = false;
+          const APOS_MERGED_BUNDLE = 'apos-bundle.js';
+          const APOS_ONLY_BUNDLE = 'apos-only-bundle.js';
+          const APOS_ONLY_TS = '.apos-only-timestamp.txt';
+          const PUBLIC_BUNDLE_CSS = 'public-bundle.css';
+          const PUBLIC_BUNDLE_JS = 'public-bundle.js';
+          let checkTimestamp = false;
+
+          if (!fs.exists(bundleDir)) {
+            rebuildAposUi = true;
+            await fs.mkdirp(bundleDir);
+          } else {
+            await fs.remove(`${bundleDir}/${APOS_MERGED_BUNDLE}`);
+            await fs.remove(`${bundleDir}/${PUBLIC_BUNDLE_CSS}`);
+            await fs.remove(`${bundleDir}/${PUBLIC_BUNDLE_JS}`);
+          }
+
+          if (process.env.CORE_DEV !== 'true') {
```

Comment:
Environment variable convention is to allow `CORE_DEV=1`, or really anything, but `1` should definitely work.